dj/sql/parsing/backends/sqloxide.py

```diff
@@ -0,0 +1,391 @@
+"""
+parsing backend turing sqloxide output into DJ AST
+"""
+from typing import List, Set, Union, cast
+
+from sqloxide import parse_sql
+
+from dj.sql.parsing.ast import (
+    Alias,
+    Between,
+    BinaryOp,
+    BinaryOpKind,
+    Boolean,
+    Case,
+    Column,
+    Expression,
+    From,
+    Function,
+    Join,
+    JoinKind,
+    Number,
+    Operation,
+    Query,
+    Select,
+    String,
+    Table,
+    UnaryOp,
+    UnaryOpKind,
+    Value,
+    Wildcard,
+)
+from dj.sql.parsing.backends.exceptions import DJParseException
+
+
+def match_keys(parse_tree: dict, *keys: Set[str]) -> bool:
+    """
+    match a dictionary having exact keys
```

Comment:
nit: I would either make the docstring more specific (`match a parse tree having exact keys`) or make the argument name more general (`d: dict`). Same for the next function.